micro_brewery/data/mrp_bom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="mrp_bom_1" model="mrp.bom">
+        <field name="product_tmpl_id" model="product.product" eval="obj().env.ref('micro_brewery.product_product_20').product_tmpl_id.id"/>
+        <field name="product_qty">10.0</field>
+        <field name="code">Blond Brewing</field>
+        <field name="produce_delay">1</field>
+        <field name="product_uom_id" ref="uom_uom_27"/>
+        <field name="byproduct_ids" model="mrp.bom.byproduct" eval="[(0, 0, {
+            'product_id': obj().env.ref('micro_brewery.product_product_10').id,
+            'product_uom_category_id': obj().env.ref('uom.product_uom_categ_kgm').id,
+            'company_id': obj().env.ref('base.main_company').id,
+            'product_qty': '15.0',
+        })]"/>
+    </record>
+    <record id="mrp_bom_4" model="mrp.bom">
+        <field name="product_tmpl_id" ref="product_template_2"/>
+        <field name="product_qty">3000.0</field>
+        <field name="code">Blond Bottling</field>
+        <field name="produce_delay">1</field>
+        <field name="type">subcontract</field>
+    </record>
+    <record id="mrp_bom_5" model="mrp.bom">
+        <field name="product_tmpl_id" ref="product_template_21"/>
+        <field name="product_qty">10.0</field>
+        <field name="code">Brown Brewing</field>
+        <field name="produce_delay">1</field>
+        <field name="product_uom_id" ref="uom_uom_27"/>
+        <field name="byproduct_ids" model="mrp.bom.byproduct" eval="[(0, 0, {
+            'product_id': obj().env.ref('micro_brewery.product_product_10').id,
+            'product_uom_category_id': obj().env.ref('uom.product_uom_categ_kgm').id,
+            'company_id': obj().env.ref('base.main_company').id,
+            'product_qty': '15.0',
+        })]"/>
```

Comment:
Here you create 3 times the byproduct, but it could be the same product if I understand correctly.
So you could define one record of `mrp.bom.byproduct` in another file, and link it in the 3 `mrp.bom` that use it.

Reply:
Why do you need to define 3 different mrp.bom.byproduct? Couldn't you link the same one thrice?

Reply:
no because byproduct_ids is One2many field